### PR TITLE
UG

### PR DESCRIPTION
**Pull request by kum-wh**
https://github.com/kum-wh/tp/blob/DG/docs/UserGuide.md

https://github.com/kum-wh/tp/blob/DG/docs/DeveloperGuide.md

### CODE REVIEW

**Comment by williamlamjy** [comment]
ps can link the ug

**Comment by theodorekwok** [comment]
is there a way to write that abort is case insensitive and must type 'abort' only without any additional characters?

File: docs/UserGuide.md [hunk 1]
```diff
@@ -405,7 +409,55 @@ Remarks: Prices will plateau out in a few days.
...
+
+```
+mTracker$edit> abort
+```
+
+**Expected outcome:**
+
+```
+mTracker$main> edit 2
+	Please enter one or more StocEtf parameters to edit separated by spaces only.
```
Comment (theodorekwok) [inline docs/UserGuide.md:449]: Got to update this

Comment (theodorekwok) [inline docs/UserGuide.md:449]:
```suggestion
	Please enter one or more Etf parameters to edit separated by a single space only.
```

**Review by theodorekwok — COMMENTED** [review]
Some minor errors

File: docs/UserGuide.md (hunk 1 above)
Comment (kum-wh) [reply docs/UserGuide.md]: I change this to stock instead, because the parameters displayed below are for stock.

**Pull request merged**